### ISSUE TITLE
Better management of incorrect arguments

### ISSUE THREAD

**Issue by sylvestre** [mozilla/grcov#245]
```
$ debian/grcov/usr/bin/grcov  foo
thread 'Producer' panicked at 'Cannot load file '"/home/sylvestre/dev/debian/debcargo-conf/build/grcov/foo"': it isn't a directory, a .info, a .json or a .xml file.', src/producer.rs:465:17
note: Run with `RUST_BACKTRACE=1` for a backtrace.

```
We could have something like:
```
Incorrect argument: 'foo' 
```

**Comment by sylvestre** [comment]
(could be a good first bug)

**Comment by marco-c** [comment]
This would be fixed by #23.
